static/js/src/static-forms.js

```diff
@@ -21,10 +21,22 @@
   });
 })();
 
-function buildCommentsForLead() {
+function buildCommentsForLead(formId) {
```

Comment:
I would attach this function dynamically on page load and then fetch the formid from the `<form>` elements `id`. This way we don't need to manually add the `onsubmit` function to each form. This will also help with future proofing as we only need to make changes in one place.